sdk/rust-examples/fd_create/src/main.rs

```diff
@@ -0,0 +1,69 @@
+// This test program is not ready to be released!
+
+#![feature(wasi_ext)]
+
+use std::fs::File;
+use std::io::prelude::*;
+use std::io::SeekFrom;
+use std::os::wasi::io::{FromRawFd, RawFd};
+
+////////////////////////////////////////////////////////////////////////////
+// This part should probably be in a Veracruz library.
```

Comment:
Can you move these into a separate support library that's linked against?

Reply:
I don't know. I could investigate at some point.

Reply:
It really ought to be possible: I don't see any problem, apart from the usual one of deciding what the crate should be called and where it should live in the file tree. It could be called `veracruz_app_lib`, for example, and live in `sdk/veracruz_app_lib`?

Reply:
That's done now.